### ISSUE TITLE
Update logo

### ISSUE THREAD

**Issue by asimonw** [emfoundation/ce100-app#1146]
The app will now be called CELLO.

To reflect this, the logo will need to be updated in:
- [x] the top left corner in the nav
- [x] on the login screen
- [x] add favicon
- [x] update notification emails (e.g. "From cello" instead of "From the CE100 team")

Asset to follow

**Comment by Cleop** [comment]
@fannyBkane - I can't remember if we're ready to go with this issue? If we are, please could you email across the relevant asset for this one? (or assets if the favicon is a variation)

**Comment by fannyBkane** [comment]
Hey @Cleop we are still waiting for legal to come back, until then we can't use or upload the logo.

**Comment by iteles** [comment]
Removing this from the current sprint as we will not have legal approval before it finishes.

**Comment by asimonw** [comment]
@Cleop Does this still have an external dependency?

**Comment by SimonLab** [comment]
@asimonw I think we now have the logos, I'll start to integrate them 👍

**Comment by asimonw** [comment]
@SimonLab I realize that the 4th item in the list is a bit vague. There are a few places where the text needs to change:

**Activation email**
Email title: "Activate your CE100 member's area account" --> "Activate your cello account"
In the body: "Welcome to the CE100 member's area!" --> "Welcome to cello!"

**Password reset email**
Email title: "Reset your password for the CE100 member's area" --> "Reset your cello password"

**Comment notification email**
In the body: "From the CE100 team" --> "From cello"

**Comment by asimonw** [comment]
@SimonLab Could you also add the new logo to the forgotten password page? It still has the old one:
<img width="490" alt="screen shot 2018-01-12 at 12 03 39" src="https://user-images.githubusercontent.com/15235804/34872310-c1f6c600-f790-11e7-8889-04c718ee5726.png">

**Comment by asimonw** [comment]
@SimonLab Same for the activation screen 😬 
<img width="739" alt="screen shot 2018-01-12 at 12 19 16" src="https://user-images.githubusercontent.com/15235804/34872839-e8eba472-f792-11e7-8d05-bc1533368db4.png">

**Comment by asimonw** [comment]
@SimonLab You must have also missed this in https://github.com/emfoundation/ce100-app/issues/1146#issuecomment-357004090

Activation email
Email title: "Activate your CE100 member's area account" --> "Activate your cello account"
In the body: "Welcome to the CE100 member's area!" --> "Welcome to cello!"

Password reset email
Email title: "Reset your password for the CE100 member's area" --> "Reset your cello password"

**Comment by asimonw** [comment]
@SimonLab The image background doesn't really work as it does in the mockup. It would work perfectly if you would remove the `min-height: 100vh` from `.container--login`. This breaks the forgotten password page, but you can instead set the `min-height: 100vh` on `.bg-login-ns`.

Also, the favicon looks pixelated on retina screens. Would it be possible to add one with 2x size targeting retina screens as well? @fannyBkane should've sent through a bunch.

That reminds me. Did you add a touch icon set? I don't see anything. They should also be in what Fanny sent through.

**Comment by asimonw** [comment]
I think @fannyBkane will still add a few comments, so you might want to wait a bit before creating a new PR. Thanks for already adding all the logos and updating the emails!

**Comment by asimonw** [comment]
Sorry bg image comment should have been for the other issue. Removed it here and will ask Cleo in the other issue.

**Comment by fannyBkane** [comment]
@SimonLab the logos look a bit fuzzy. I have emailed you two new files (for nav bar and login page) - could you replace?

**Comment by asimonw** [comment]
@SimonLab to summarise: the only outstanding items are
+ add favicon for retina screens and a range of touch icons (you can provide a lot, but I think adding a set of touch icons to cover the standard range would be enough)
+ replace the logos with the new ones sent by Fanny in the navbar and on the login/activation/password-rest screens

Would that still be possible today?

**Comment by SimonLab** [comment]
@asimonw I've just created PR which change the logos and add all the favicons, it should be ready to test soon once the tests are passed and the PR is merged

**Comment by fannyBkane** [comment]
Hey @SimonLab I am afraid that the logo on the login page looks cropped, and it actually has to do with the file. Arg. Can you switch to a new png (see email)? This last one should work!

**Comment by SimonLab** [comment]
@fannyBkane the logo looks ok for me:
![image](https://user-images.githubusercontent.com/6057298/34884141-bb164702-f7b3-11e7-9efd-158ca587a401.png)
Which device are you testing on? Maybe I'm missing something too, but I don't mind to change to a new logo version 👍

**Comment by SimonLab** [comment]
@fannyBkane @asimonw this should be ready for testing

**Comment by asimonw** [comment]
Me too! Closing 🎉
